pallets/stableswap/src/lib.rs

```diff
@@ -281,6 +294,9 @@ pub mod pallet {
 
 		/// Not allowed to perform an operation on given asset.
 		NotAllowed,
+
+		/// Future block number is in the past.
+		InvalidBlock,
```

Comment:
according to the error description provided, error could be named for example `PastBlock` instead of general `Invalid`